2022-01-12 09:42:26,778 INFO    MainThread:4928 [wandb_setup.py:_flush():69] setting env: {}
2022-01-12 09:42:26,779 INFO    MainThread:4928 [wandb_setup.py:_flush():69] setting login settings: {}
2022-01-12 09:42:26,779 INFO    MainThread:4928 [wandb_init.py:_log_setup():336] Logging user logs to /workspace/graph-neural-pde/src/wandb/run-20220112_094226-16bclyy1/logs/debug.log
2022-01-12 09:42:26,779 INFO    MainThread:4928 [wandb_init.py:_log_setup():337] Logging internal logs to /workspace/graph-neural-pde/src/wandb/run-20220112_094226-16bclyy1/logs/debug-internal.log
2022-01-12 09:42:26,780 INFO    MainThread:4928 [wandb_init.py:init():369] calling init triggers
2022-01-12 09:42:26,780 INFO    MainThread:4928 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2022-01-12 09:42:26,780 INFO    MainThread:4928 [wandb_init.py:init():418] starting backend
2022-01-12 09:42:26,780 INFO    MainThread:4928 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-01-12 09:42:26,826 INFO    MainThread:4928 [backend.py:ensure_launched():123] starting backend process...
2022-01-12 09:42:26,834 INFO    MainThread:4928 [backend.py:ensure_launched():128] started backend process with pid: 4978
2022-01-12 09:42:26,835 INFO    MainThread:4928 [wandb_init.py:init():423] backend started and connected
2022-01-12 09:42:26,836 INFO    MainThread:4928 [wandb_init.py:init():465] updated telemetry
2022-01-12 09:42:26,836 INFO    MainThread:4928 [wandb_init.py:init():484] communicating current version
2022-01-12 09:42:27,522 INFO    MainThread:4928 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-12 09:42:27,523 INFO    MainThread:4928 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2022-01-12 09:42:28,091 INFO    MainThread:4928 [wandb_init.py:init():522] starting run threads in backend
2022-01-12 09:42:31,550 INFO    MainThread:4928 [wandb_run.py:_console_start():1524] atexit reg
2022-01-12 09:42:31,550 INFO    MainThread:4928 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2022-01-12 09:42:31,551 INFO    MainThread:4928 [wandb_run.py:_redirect():1376] Redirecting console.
2022-01-12 09:42:31,552 INFO    MainThread:4928 [wandb_run.py:_redirect():1438] Redirects installed.
2022-01-12 09:42:31,552 INFO    MainThread:4928 [wandb_init.py:init():546] run started, returning control to user process
2022-01-12 09:42:38,403 INFO    MainThread:4928 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2022-01-12 09:42:38,404 INFO    MainThread:4928 [wandb_run.py:_restore():1466] restore
2022-01-12 09:42:40,918 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 837
  total_bytes: 837
}

2022-01-12 09:42:41,129 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 837
  total_bytes: 6889
}

2022-01-12 09:42:41,671 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 837
  total_bytes: 6889
}

2022-01-12 09:42:41,775 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 837
  total_bytes: 6889
}

2022-01-12 09:42:41,878 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6889
  total_bytes: 6889
}

2022-01-12 09:42:41,980 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6889
  total_bytes: 6889
}

2022-01-12 09:42:42,083 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6889
  total_bytes: 6889
}

2022-01-12 09:42:42,193 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6889
  total_bytes: 6889
}

2022-01-12 09:42:42,296 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6889
  total_bytes: 6889
}

2022-01-12 09:42:42,400 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6889
  total_bytes: 6889
}

2022-01-12 09:42:42,503 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6889
  total_bytes: 6889
}

2022-01-12 09:42:42,605 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6889
  total_bytes: 6889
}

2022-01-12 09:42:42,706 INFO    MainThread:4928 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6889
  total_bytes: 6889
}

2022-01-12 09:42:43,905 INFO    MainThread:4928 [wandb_run.py:_show_files():1838] logging synced files
